src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -506,19 +506,24 @@ trait DeploymentFormats {
     }
   )
 
+  def actionInstanceOn(runSpec: RunSpec): String = runSpec match {
```

Comment:
nit: would like a better name for this at some point. some suggestions:
- `actionInstanceKind`
- `instanceKind`
- `kind`

the current name tells me very little